packages/plugins/common/src/react.tsx

```diff
@@ -1,4 +1,5 @@
 import { AllTypes, definePlugin } from '@zodui/core'
+import { flatUnwrapUnion } from '@zodui/core/utils'
```

Comment:
```ts
import { flatUnwrapUnion } from './utils'
```
Import the util function by relative path that can help bundler to resolve the dependencies.